### PR TITLE
SPORTSHUB 137 - Fixed the issue with two instances of is your event private/public on…

### PR DESCRIPTION
**Pull request by ReggieStar11**
… the intial screen of create event

## Why was your change needed?

Please detail here why you did the change you did.

## What was Changed?

Please detail here what were the main changes you made.

## Any extra notes for reviewers to know?

Write any extra notes you may have for the PR.
This may include specific code paths that need extra scrutiny, or how to review a specific portion of code.

### CODE REVIEW

File: frontend/components/events/create/forms/BasicForm.tsx [hunk 1]
```diff
@@ -254,28 +233,27 @@ export function BasicInformation({
...
-                    const privacyValue = e || "Public";
-                    handlePrivacyChange(privacyValue);
-                  }}
-                >
-                  <Option value="Public">Public</Option>
-                  <Option value="Private">Private</Option>
-                </Select>
-              </div>
+          </div>
+          <div>
```
Comment (owya490) [inline frontend/components/events/create/forms/BasicForm.tsx:237]: is this full width or the half width one?

**Review by owya490 — APPROVED** [review]
thanks reggie, i think i added the bug :<

File: frontend/components/events/create/forms/BasicForm.tsx (hunk 1 above)
Comment (ReggieStar11) [reply frontend/components/events/create/forms/BasicForm.tsx:237]: full width

**Comment by ReggieStar11** [comment]
> thanks reggie, i think i added the bug :<

easy fix dw

**Pull request merged**